tests/test_other.py

```diff
@@ -3595,6 +3595,10 @@ def test_dashM_respect_dashO(self):
     assert len(with_dash_o) == 0
     assert len(without_dash_o) != 0
 
+  def test_null_device_as_input(self):
+    out = Popen([PYTHON, EMXX, '-E', '-dM', '-xc', '/dev/null'], stdout=PIPE).communicate()[0]
```

Comment:
I think the test should do something different on windows than on other platforms?